demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -107,19 +130,34 @@ def run(params, capture, detector, reid):
         with open(params.history_file, 'w') as outfile:
             json.dump(history, outfile)
 
+    if len(config['embeddings']['save_path']):
+        save_embeddings(tracker.scts, **config['embeddings'])
+
 
 def main():
     """Prepares data for the person recognition demo"""
     parser = argparse.ArgumentParser(description='Multi camera multi person \
                                                   tracking live demo script')
     parser.add_argument('-i', type=str, nargs='+', help='Input sources (indexes \
                         of cameras or paths to video files)', required=True)
+    parser.add_argument('--detections', type=str, nargs='+',
+                        help='Json files with detections')
 
     parser.add_argument('-m', '--m_detector', type=str, required=True,
                         help='Path to the person detection model')
     parser.add_argument('--t_detector', type=float, default=0.6,
                         help='Threshold for the person detection model')
 
+    parser.add_argument('--m_segmentation', type=str, required=False,
+                        help='Path to the instance segmentation model')
```

Comment:
```suggestion
                        help='Path to the person instance segmentation model')
```

Seems like it should be consistent with the next option.

Reply:
Fixed.